src/nvim/log.c

```diff
@@ -76,8 +76,8 @@ static bool log_path_init(void)
     if (!os_isdir((char_u *)cachehome)) {
         int ret;
         if ((ret = os_mkdir_recurse(cachehome, 0700, &failed_dir)) != 0) {
-          EMSG3(_(LOGERR "Failed to create directory %s "
-                  "for writing logs: %s"),
+          WLOG(LOGERR "Failed to create directory %s "
+                  "for writing logs: %s",
                 failed_dir, os_strerror(ret));
```

Comment:
Since it's not displayed to the user, there's no need for an error-code (so that can be removed from the docs, too).

Also, the formatting good be a little more compact, and lint wants `failed_dir` to line up with the column after the opening `(`.

```suggestion
          WLOG("Failed to create directory %s for writing logs: %s",
               failed_dir, os_strerror(ret));
```